department-of-veterans-affairs/abd-vro
# Introduce single page apps.

## Summary

Set up structure for single page apps.

Associated tickets or Slack threads:
- https://github.com/department-of-veterans-affairs/abd-vro/issues/3554

## How to test this PR
1. Pull branch
2. from root dir, run `./gradlew vro-streamlit:check` to run all checks/lints/unit tests.
3. from root dir, run `./gradlew :vro-streamlit:docker ` then run `COMPOSE_PROFILES="streamlit" ./gradlew :dockerComposeUp` to run the container. Then open browser to http://0.0.0.0:8501/ to view webpages.
